packages/orchestration/src/cosmos-api.ts

```diff
@@ -77,6 +77,11 @@ export type CosmosChainInfo = {
    */
   allowedMessages: TypeUrl[];
   allowedQueries: TypeUrl[];
+
+  /**
+   * cf https://github.com/cosmos/chain-registry/blob/master/chain.schema.json#L117
+   */
+  stakingTokens?: [{ denom: string }];
```

Comment:
That syntax gave me a strange error about `[X]` vs. `X[]`. This made it go away:

```suggestion
  stakingTokens?: Array<{ denom: string }>;
```

Reply:
I imagine that's your linter. That is a better type though, because what's here is a tuple rather than an array of arbitrary length. I'll change it.